pkg/services/authn/authnimpl/sync/user_sync.go

```diff
@@ -184,7 +184,7 @@ func (s *UserSync) upsertAuthConnection(ctx context.Context, userID int64, ident
 	if createConnection {
```

Comment:
Hey @mgyongyosi will this change start to create user_auth entries for basic auth and api key auth? This might be problematic but not an active issue.

Most important would be for sessions to not set auth modules as they should take the 'last authenticated' method

Reply:
Nope, because the `SyncHook` is only executed when the `ClientParams.SyncUser` is set to `true` and it's set to true for only external auth.

Reply:
works for me, thanks for the confirmation